lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -159,6 +159,7 @@
   links:
     description:
       - List of name aliases for linked containers in the format C(container_name:alias)
+        Setting this will force container to be restarted.
```

Comment:
I would make it a second item in that list.
Since this is a proposal through the Github interface I will make that change myself, but leave it up to the Docker people whether this is correct.

Reply:
Thanks